build.gradle

```diff
@@ -269,7 +269,7 @@ model {
 
         all {
             binaries.all {
-                if (targetPlatform.operatingSystem.name in ['linux', 'freebsd'] && targetPlatform.architecture != arch) {
+                if (targetPlatform.operatingSystem.name in ['linux', 'freebsd'] && targetPlatform.architecture != arch && !(arch.amd64 && targetPlatform.architecture.i386)) {
```

Comment:
I don't think we should just assume that we can build the x86 variant, because often the required dependencies are missing. Some options:

- probe for the library
- fix the native plugins to probe for the library
- add some opt-in flag
- assume, but probe for the library and fail if not present